frameworks/kafka/tests/test_utils.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+
+import json
+import dcos
+import shakedown
+import dcos.config
+import dcos.http
+import sdk_spin as spin
+
+from tests.config import (
+    PACKAGE_NAME,
+    DEFAULT_BROKER_COUNT
+)
+
+
+def service_cli(cmd_str):
+    full_cmd = '{} {}'.format(PACKAGE_NAME, cmd_str)
+    ret_str = command.run_cli(full_cmd)
+    return json.loads(ret_str)
+
+
+def check_health():
+    def fn():
+        try:
+            return shakedown.get_service_tasks(PACKAGE_NAME)
+        except dcos.errors.DCOSHTTPException:
+            return []
+
+    def success_predicate(tasks):
+        running_tasks = [t for t in tasks if t['state'] == TASK_RUNNING_STATE]
+        print('Waiting for {} healthy tasks, got {}/{}'.format(
+            DEFAULT_BROKER_COUNT, len(running_tasks), len(tasks)))
+        return (
+            len(running_tasks) == DEFAULT_BROKER_COUNT,
+            'Service did not become healthy'
+        )
+
+    return spin.spin(fn, success_predicate)
+
+
+def get_running_broker_task(broker_name):
+    def fn():
+        try:
+            tasks = shakedown.get_service_tasks(PACKAGE_NAME)
+            return [t for t in tasks if t['state'] == 'TASK_RUNNING' and t['name'] == broker_name]
+        except dcos.errors.DCOSHTTPException:
+            return []
+
+    def success_predicate(tasks):
+        return len(tasks) == 1, 'Failed to get task'
+
+    return spin.spin(fn, success_predicate)
+
+
+def task_id_changes(broker_name, task_id):
```

Comment:
You might want to remove this and use the existing function check_tasks_updated(): https://github.com/mesosphere/dcos-commons/blob/master/testing/sdk_tasks.py#L37